src/main/java/org/automation/pageObjects/DirectorPage.java

```diff
@@ -5,111 +5,157 @@
 import org.automation.utilities.Assertions;
 import org.automation.utilities.WebdriverWaits;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static org.automation.utilities.Assertions.validate_text;
 
 
 public class DirectorPage extends BasePage {
-    WebdriverWaits wait=new WebdriverWaits();
-    public By directorsTab= By.xpath("//li[@id='Directors']");
-    public By createDirectorButton=By.xpath("//button[text()='Create Director']");
-    public By directorsFirstName=By.xpath("//input[@placeholder='First Name']");
-    public By directorsLastName=By.xpath("//input[@placeholder='Last Name']");
-    public By directorsMobileNumber=By.xpath("//input[@placeholder='(999) 999-9999']");
-    public By directorsEmail=By.xpath("//input[@placeholder='Email']");
-    public By assignLocation=By.xpath("//select[@id='testingLocation']");
-    public By directorsLocationName=By.xpath("//option[text()='Plano']");
-    public By directorsUserName=By.xpath("//input[@placeholder='Username']");
-    public By password_Field=By.xpath("//input[@placeholder='Create Password']");
-    public By confirm_PasswordField=By.xpath("//input[@placeholder='Confirm Password']");
-    public By createDirectorsButton=By.xpath("//button[text()='Create Director']");
+    WebdriverWaits wait = new WebdriverWaits();
```

Comment:
In between the xpath, why we are creating the instance of class?

Reply:
I have verified and that instance of class is not exist in current codes